cpp/velox/compute/VeloxBackend.cc

```diff
@@ -116,6 +119,12 @@ void VeloxBackend::printConf(const std::unordered_map<std::string, std::string>&
 }
 
 void VeloxBackend::init(const std::unordered_map<std::string, std::string>& conf) {
+  // Init glog and verbose level.
+  auto vlogLevel = stoi(getConfigValue(conf, kGlogVerboseLevel, kGlogVerboseLevelDefault));
```

Comment:
We need to send this config from Scala side as [GlutenConfig.scala#L410-L458](https://github.com/oap-project/gluten/blob/main/shims/common/src/main/scala/io/glutenproject/GlutenConfig.scala#L410-L458), then it can extracted here, right?

Reply:
I have test this patch in my local env, no problem.

Reply:
Thanks for checking.